.circleci/config.yml

```
@@ -1890,8 +1928,14 @@ workflows:
       # build-only
       - b_docs: *requires_nothing
       - b_ubu_ossfuzz: *requires_nothing
-      - b_ubu_2204: *requires_nothing
-      - b_ubu_2204_clang: *requires_nothing
+      - b_ubu_min_req_clang: *requires_nothing
+
+      # build and test on ubuntu 22.04
+      - b_ubu_min_req: *requires_nothing
```

It's no longer 22.04:
```suggestion
      # build and test with minimum supported versions of dependencies
      - b_ubu_min_req_clang: *requires_nothing
      - b_ubu_min_req: *requires_nothing
```